etc/sync-function-template.js

```diff
@@ -287,7 +287,7 @@ function synctos(doc, oldDoc) {
           }
           break;
         case 'object':
-          if (typeof itemValue !== 'object') {
+          if (typeof itemValue !== 'object' || itemValue instanceof Array) {
```

Comment:
Addresses https://github.com/Kashoo/synctos/issues/42 for the "object" validation type.